Missionframework/scripts/shared/fetch_params.sqf

```diff
@@ -14,124 +16,126 @@ KP_liberation_kill_debug = ["DebugKill",0] call bis_fnc_getParamValue;
-GRLIB_halo_param = ["HaloJump",1] call F_getSaveableParam;
-GRLIB_use_whitelist = ["Whitelist",0] call F_getSaveableParam;
-GRLIB_cleanup_vehicles = ["CleanupVehicles",2] call F_getSaveableParam;
-GRLIB_csat_aggressivity = ["Aggressivity",2] call F_getSaveableParam;
-GRLIB_weather_param = ["Weather",3] call F_getSaveableParam;
-GRLIB_shorter_nights = ["ShorterNights",0] call F_getSaveableParam;
-GRLIB_remote_sensors = 0;
-GRLIB_blufor_defenders = ["BluforDefenders",1] call F_getSaveableParam;
-GRLIB_autodanger = ["Autodanger",0] call F_getSaveableParam;
-GRLIB_maximum_fobs = ["MaximumFobs",26] call F_getSaveableParam;
-GRLIB_max_squad_size = ["MaxSquadSize",10] call F_getSaveableParam;
-KP_liberation_arsenalUsePreset = ["ArsenalUsePreset",1] call F_getSaveableParam;
-KP_liberation_mapmarkers = ["MapMarkers",1] call F_getSaveableParam;
-KP_liberation_mobilerespawn = ["MobileRespawn",1] call F_getSaveableParam;
-KP_liberation_mobilearsenal = ["MobileArsenal",1] call F_getSaveableParam;
-KP_liberation_ailogistics = ["AiLogistics",1] call F_getSaveableParam;
-KP_liberation_restart = ["ServerRestart",0] call F_getSaveableParam;
-KP_liberation_cr_param_buildings = ["CR_Building",0] call F_getSaveableParam;
-KP_liberation_respawn_cooldown = ["RespawnCooldown",900] call F_getSaveableParam;
-KP_liberation_clear_cargo = ["ClearCargo",1] call F_getSaveableParam;
-
-if (GRLIB_fatigue < 0.1) then {GRLIB_fatigue = false} else {GRLIB_fatigue = true};
-if (GRLIB_introduction == 1) then {GRLIB_introduction = true} else {GRLIB_introduction = false};
-if (GRLIB_deployment_cinematic == 1) then {GRLIB_deployment_cinematic = true} else {GRLIB_deployment_cinematic = false};
-if (GRLIB_build_first_fob == 1) then {GRLIB_build_first_fob = true} else {GRLIB_build_first_fob = false};
-if (GRLIB_teamkill_penalty == 1) then {GRLIB_teamkill_penalty = true} else {GRLIB_teamkill_penalty = false};
-if (GRLIB_adaptive_opfor == 1) then {GRLIB_adaptive_opfor = true} else {GRLIB_adaptive_opfor = false};
-if (GRLIB_permissions_param == 1) then {GRLIB_permissions_param = true} else {GRLIB_permissions_param = false};
-if (GRLIB_use_whitelist == 1) then {GRLIB_use_whitelist = true} else {GRLIB_use_whitelist = false};
-if (GRLIB_shorter_nights == 1) then {GRLIB_shorter_nights = true} else {GRLIB_shorter_nights = false};
-if (GRLIB_blufor_defenders == 1) then {GRLIB_blufor_defenders = true} else {GRLIB_blufor_defenders = false};
-if (GRLIB_autodanger == 1) then {GRLIB_autodanger = true} else {GRLIB_autodanger = false};
-if (KP_liberation_arsenalUsePreset == 1) then {KP_liberation_arsenalUsePreset = true} else {KP_liberation_arsenalUsePreset = false};
-if (KP_liberation_mapmarkers == 1) then {KP_liberation_mapmarkers = true; GREUH_allow_mapmarkers = true; GREUH_allow_platoonview = true} else {KP_liberation_mapmarkers = false; GREUH_allow_mapmarkers = false; GREUH_allow_platoonview = false; show_platoon = false; show_teammates = false; show_nametags = false};
```

Comment:
I have removed `show_platoon = false; show_teammates = false; show_nametags = false` completely from `fetch_params` as these variables are defaulted to false anyway. Also script using these variables will not be `spawn`ed when `GREUH_allow_platoonview` is set to `false`.